app/models/goal.py

```diff
@@ -1,5 +1,20 @@
 from app import db
 
-
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement = True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", back_populates = "goal") # is a list
+
+    def to_dict_goal(self):
+        return {"goal": {
+        "id":self.goal_id, 
+        "title":self.title,
+        }}
```

Comment:
This could be DRY-ed slightly:
```suggestion
    def to_dict_goal(self):
        return {"goal": self.to_dict_all_goals()}
```